### PR TITLE
BLOG-210: fix the boolean column filter in SQL

### PR DESCRIPTION
**Pull request by zenihex**
https://jira.xwiki.org/projects/BLOG/issues/BLOG-210?filter=allopenissues

### CODE REVIEW

File: application-blog-ui/src/main/resources/Blog/EnablePanelsConfigurationClass.xml [hunk 1]
```diff
@@ -72,7 +72,7 @@
       <separators/>
       <size>1</size>
       <sort>none</sort>
-      <sql>select doc.fullName from XWikiDocument doc, BaseObject obj where obj.name = doc.fullName and obj.className = 'Blog.BlogClass' and doc.fullName&lt;&gt;'Blog.BlogTemplate' #if(($xwiki.getUserPreference('displayHiddenDocuments') == 0) || ("$!services.user.properties" != '' &amp;&amp; !$services.user.properties.displayHiddenDocuments())) and doc.hidden&lt;&gt;1 #end</sql>
+      <sql>select doc.fullName from XWikiDocument doc, BaseObject obj where obj.name = doc.fullName and obj.className = 'Blog.BlogClass' and doc.fullName&lt;&gt;'Blog.BlogTemplate' #if(($xwiki.getUserPreference('displayHiddenDocuments') == 0) || ("$!services.user.properties" != '' &amp;&amp; !$services.user.properties.displayHiddenDocuments())) and not doc.hidden #end</sql>
```
Comment (tmortagne) [inline application-blog-ui/src/main/resources/Blog/EnablePanelsConfigurationClass.xml:75]: Have you tested this on MySQL ?

**Comment by zenihex** [comment]
image

File: application-blog-ui/src/main/resources/Blog/EnablePanelsConfigurationClass.xml (hunk 1 above)
Comment (tmortagne) [reply application-blog-ui/src/main/resources/Blog/EnablePanelsConfigurationClass.xml:75]: Sounds, good, thanks !

**Comment by zenihex** [comment]
> To be sure: @zenihex what's your id on jira.xwiki.org ?

my userid on jira.xwiki.org is zenihex@gmail.com. Thanks.

**Comment by tmortagne** [comment]
Thanks, a 9.13.4 version is now available with your fix.

**Comment by zenihex** [comment]
There might be a bug in the fix:

`not doc.hidden` => `is not doc.hidden`

**Pull request merged**